index.html

```diff
@@ -4,464 +4,524 @@
-                class="right-wrapper__ellipse-small"
-                src="./src/img/ellipse-small.svg"
-                alt="small ellipse"
-              />
+      <!-- Hamburger -->
+      <button class="hamburger">
+        <span class="hamburger__item"></span>
+        <span class="hamburger__item"></span>
+        <span class="hamburger__item"></span>
+      </button>
+
+      <div class="container header-wrapper">
+        <a class="logo link-black" href="#">
+          J<span class="decor-orange">o</span>bply
+        </a>
+
+        <!-- Navigation menu -->
+        <nav class="menu">
+          <!-- Navigation list items -->
+          <ul class="menu__list">
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#">Home</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#">Categories</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#">Service</a>
+            </li>
+            <li class="menu__list-item list-item">
+              <a class="nav-link" href="#">Jobs</a>
+            </li>
+          </ul>
+        </nav>
+
+        <!-- Navigation button -->
+        <button class="buttons buttons__cta-button" href="#">
+          Post a vacancy
+        </button>
+      </div>
+    </header>
 
-              <!-- Flying icons -->
-              <div class="right-wrapper__flying-icon google-icon">
+    <main>
+      <!-- Hero section -->
+      <section class="hero">
+        <!-- Hero section wrapper -->
+
+        <!-- Info -->
+        <div class="info">
+          <h1 class="info__title title">Find & Apply For Your Dream Job</h1>
+          <p class="info__subtitle">
+            The website is chockablock with useful features, allowing you to
+            efficiently search and apply for roles across all sectors,
+            experience levels and locations.
+          </p>
+
+          <!-- Search container -->
+          <div class="search-container">
+            <!-- Search form -->
+            <form class="search-container__search-form" method="get">
+              <div class="search-area">
                 <img
-                  class="google-img"
-                  src="./src/icons/flat-color-icons_google.svg"
-                  alt="google icon"
+                  class="search-container__search-icon"
+                  src="./src/icons/search-icon.svg"
+                  alt="search"
                 />
-              </div>
 
-              <div class="right-wrapper__flying-icon paypal-icon">
-                <img
-                  class="paypal-img"
-                  src="./src/icons/logos_paypal.svg"
-                  alt="paypal icon"
+                <input
+                  class="search-input"
+                  type="text"
+                  placeholder="Job title or Keyword"
+                  name="text"
+                  value=""
```

Comment:
```suggestion
                  placeholder="Job title or Keyword"
                  name="text"
```

No need in value attribute as you don't set any default value

Reply:
Delete value attribute